src/formatters/mdapi/retrieveResultFormatter.ts

```diff
@@ -59,7 +49,7 @@ export class RetrieveResultFormatter extends ResultFormatter {
    * @returns RetrieveCommandResult
    */
   public getJson(): RetrieveCommandResult {
-    return this.result;
+    return Object.assign({}, this.result, { zipFilePath: this.zipFilePath }) as RetrieveCommandResult;
```

Comment:
```suggestion
    return { ...this.result, zipFilePath: this.zipFilePath };
```

Reply:
Yeah that's better, thanks! I'll update.